lib/utils/copy-dir.js

```diff
@@ -14,10 +14,15 @@ module.exports = function copyDir(opts) {
   console.log(messages.copying(projectName));
 
   return new Promise(function(resolve, reject) {
-    const stopCopySpinner = output.wait('Copying files');
+    const stopCopySpinner = output.wait("Copying files");
 
-    fs
-      .copy(templatePath, projectPath)
+    fs.copy(templatePath, projectPath)
+      .then(function() {
+        return fs.move(
+          path.resolve(projectPath, "./.gitignore"),
```

Comment:
Moving the same file into the same file?

Reply:
When it's packaged, the .gitignore will be gone, so you have to have it (in volto-starter-kit) as gitignore (as the CRA guys does), then move it once the folder is copied over